geomesa-web/geomesa-web-data/pom.xml

```diff
@@ -80,7 +80,7 @@
             <dependency>
                 <groupId>com.beust</groupId>
                 <artifactId>jcommander</artifactId>
-                <version>1.32</version>
+                <version>1.48</version>
```

Comment:
note: 1.48 is already approved by locationtech: https://dev.eclipse.org/ipzilla/show_bug.cgi?id=10140